repos/system_upgrade/el7toel8/actors/persistentnetnamesdisable/actor.py

```diff
@@ -1,10 +1,11 @@
 import re
 
-from leapp.libraries.common import reporting
 
 from leapp.actors import Actor
 from leapp.models import PersistentNetNamesFacts, KernelCmdlineArg
 from leapp.tags import FactsPhaseTag, IPUWorkflowTag
+from leapp.reporting import Report
+from leapp.libraries.common import reporting
```

Comment:
Sort of nit picking, but do you mind sorting the leapp imports alphabetically?

Reply:
I can.